packages/mdc-select/foundation.js

```diff
@@ -105,9 +105,6 @@ export default class MDCSelectFoundation extends MDCFoundation {
     };
     this.cancelHandler_ = () => {
       this.close_();
-      if (this.selectedIndex_ === -1) {
```

Comment:
This is needed to have the floating label class removed when the menu is closed.